### PR TITLE
Redesign openLoan UI

### PR DESCRIPTION
**Pull request by Pochetes**
## Overview
- implemented different collateral type outputs with user input
- added basic loan metrics such as interest rate, health factor, etc.
- switched from using a modal to a dropdown for choosing collateral type
- cleaned and refactored a lot of unecessary code

### Demo
providing lp tokens as collateral type
Screen Shot 2022-09-28 at 2 32 58 AM

providing one of the tokens as collateral type
Screen Shot 2022-09-28 at 2 33 51 AM

providing both tokens as collateral type
Screen Shot 2022-09-28 at 2 34 12 AM

**NOTE**: I made some comments so that Usama or whoever is working on this knows that it needs to be worked on. Some of the logic for the `openLoanHandler` is a little messy, therefore we need to provide some clarification on that.

### CODE REVIEW

File: src/components/OpenLoan/TokenUserInput.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  tokenPairBalance: 'font-semibold text-xxs text-neutrals-600 tracking-wide text-right mt-2',
+}
+
+interface TokenUserInputProps {
+  collateralType: string
+  token0: Token
+  token1: Token
+}
+
+export const TokenUserInput = ({ collateralType, token0, token1 }: TokenUserInputProps) => {
```
Comment (TBomberman) [inline src/components/OpenLoan/TokenUserInput.tsx:24]: Is this component reused? If not, possibly rename to CollateralUserInput.

**Review by TBomberman — APPROVED** [review]

File: src/components/OpenLoan/TokenUserInput.tsx (hunk 1 above)
Comment (Pochetes) [reply src/components/OpenLoan/TokenUserInput.tsx:24]: I was thinking of making it reusable and just passing the tokens and a type (whether with one token or both). However, that would increase the complexity of how to add support for the other component so for now I'll keep it for collateral type use.

File: pages/positions/openloan.tsx [hunk 2]
```diff
@@ -24,7 +77,19 @@ const tips = {
     'penalty) as a bonus.'
 }
 
-const OpenLoanPage: NextPage = () => {
+// TODO: need to figure whether to use this or enums
```
[the remark this answers is not in the document]
Comment (Pochetes) [reply pages/positions/openloan.tsx:80]: Okay, I will look into that further when we implement the functionality of `openLoan`.

File: src/components/OpenLoan/TokenUserInput.tsx (hunk 1 above)
Comment (Pochetes) [reply src/components/OpenLoan/TokenUserInput.tsx:24]: done

**Pull request merged**